### PR TITLE
changed folder name to workspace everywhere

### PR DESCRIPTION
**Pull request by asdftamir**

### CODE REVIEW

File: aws-traffic-mirroring/templates/seekret-sniffer.yaml [hunk 1]
```diff
@@ -33,10 +33,10 @@ Parameters:
   BucketSecretKey:
     Type: String
     Description: Secret key for the bucket
-  BucketFolderName:
+  Workspace:
```
Comment (guyarb) [inline aws-traffic-mirroring/templates/seekret-sniffer.yaml:36]: change the documentation of the traffic mirroring

File: docker/conf.env [hunk 2]
```diff
@@ -2,7 +2,7 @@ ACCESS_KEY=
 SECRET_ACCESS_KEY=
 STORAGE_PROVIDER=
 BUCKET_NAME=
-FOLDER_NAME=
+WORKSPACE=
```
Comment (guyarb) [inline docker/conf.env:5]: fix in the docs as well

**Review by guyarb — DISMISSED** [review]

File: aws-traffic-mirroring/templates/seekret-sniffer.yaml (hunk 1 above)
Comment (val06) [reply aws-traffic-mirroring/templates/seekret-sniffer.yaml:36]: wasn't mentioned in the readme at the first place

**Review by val06 — APPROVED** [review]

File: docker/conf.env (hunk 2 above)
Comment (val06) [reply docker/conf.env:5]: already fixed in previous PR

**Pull request merged**